### ISSUE TITLE
Automate deployment via CloudFormation

### ISSUE THREAD

**Issue by ryanchard** [funcx-faas/funcx-web-service#171]
We need to encode the web service deployment as a cloudformation script so we can easily bring up new deployments and recover from failures.

**Comment by yadudoc** [comment]
We have three major components, the frontend managed with elastic beanstalk, the forwarder and forwarder service running on a a tiny node, and the the redis and RDS databases that act as the glue between these. The component that updates the most is the frontend and that already has a bunch of workflow support in beanstalk. Marginal utility in converting to cloudformation now is limited. If we do migrate from a personal to project account, that might be the point to bite the bullet on this.

**Comment by ryanchard** [comment]
I agree. Configuring and linking all the IAMs/security groups/VPCs etc. in cloudformation is a fair bit of work. We should also wait until we land on a long-term approach for managing forwarders. If we switch to EKS or something it'd simplify the deployment process.
